internal/crdao/types/dao_contracts_metadata.go

```diff
@@ -0,0 +1,136 @@
+package types
```

Comment:
It's not a `type` we reuse but rather a value holder definition. Maybe let's move it under the `dao/` package. We probably could drop `events` in there as well.